apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1858,6 +1883,76 @@ static void libc_stdio_fileno_tc(void)
 }
 
 /**
+* @testcase         libc_stdio_lib_rdflush_tc
+* @brief            Flush read data from the I/O buffer and adjust the file pointer to account for the unread data.
+* @scenario         Flush read data from the I/O buffer and adjust the file pointer to account for the unread data.
+* @apicovered       lib_rdflush
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_lib_rdflush_tc(void)
+{
+	char *filename = VFS_FILE_PATH;
+	FILE *stream;
+	int ret;
+
+	stream = fopen(filename, "r");
+	TC_ASSERT_NEQ("fopen", stream, NULL);
+
+	ret = lib_rdflush(stream);
+	fclose(stream);
+	TC_ASSERT_EQ("lib_rdflush", ret, OK);
+
+	TC_SUCCESS_RESULT();
+}
+
+#ifdef CONFIG_STDIO_LINEBUFFER
+/**
+* @testcase         libc_stdio_lib_snoflush_tc
+* @brief            provides a common, dummy flush method for seekable output streams that are not flushable.
+* @scenario         Only used if CONFIG_STDIO_LINEBUFFER is selected.
+* @apicovered       lib_snoflush
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_lib_snoflush_tc(void)
+{
+	int ret;
+	struct lib_sostream_s *stream = stdout;
+
+	ret = lib_snoflush((FAR struct lib_sostream_s *)&stream);
+	TC_ASSERT_EQ("lib_snoflush", ret, OK);
+
+	TC_SUCCESS_RESULT();
+}
+#endif
+
+/**
+* @testcase         libc_stdio_lib_sprintf_tc
+* @brief            Composes a string with the same text that would be printed if format was used on printf
+*                   But instead of being printed the content is stored as a C string in the buffer pointed by str
+* @scenario         A terminating null character is automatically appended after the content.
+* @apicovered       lib_sprintf
+* @precondition     The size of the buffer should be large enough to contain the entire resulting string
+* @postcondition    NA
+*/
+static void libc_stdio_lib_sprintf_tc(void)
+{
+	struct lib_memoutstream_s memoutstream;
+	char *str = VFS_TEST_CONTENTS_1;
+	FAR char buf[BUFLEN];
+	int ret;
+
+	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buf, BUFLEN);
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buf));
+
+	ret = lib_sprintf((FAR struct lib_outstream_s *)&memoutstream, "%s", str);
```

Comment:
I think str variable is not needed.
you can put VFS_TEST_CONTENTS_1 here or change strlen(str) below.